webpack/published-ts-components.js

```diff
@@ -3,7 +3,7 @@ const publishedTSComponents = {
   // plop_marker:published-components
   Badge: "components/Badge/Badge",
   Text: "components/Text/Text",
-  Title: "components/Title/Title",
+  Title: "components/Heading/Heading",
```

Comment:
It's still under Title name here? Is that ok?

Reply:
Perhaps it should be mentioned somewhere?

Reply:
fixed